content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -28,7 +28,7 @@ You must use a kubectl version that is within one minor version difference of yo
 1. Download the latest release with the command:
 
     ```
-    curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
```

Comment:
Line 34 implies that the correct form is:
```suggestion
    curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
```

Reply:
Thank you, It actually seems that I pasted in the wrong command. I blame it on being on holidays for the last 3 weeks ;)